docs/API.md

```diff
@@ -481,7 +481,7 @@ Type: `Array`, `String` or `Function`
 A task name, a function or an array of either.
 
 
-### gulp.watch(glob[, opts], fn)
+### gulp.watch(glob[, opts][, fn])
```

Comment:
No, because that implies that if you want to specify `fn`, you have to specify `opts`, and that's not true. They're independently optional and so `glob[, opts][, fn]` is correct :)